### DISCUSSION TITLE
Welcome to Plex-Audiobook-Guide Discussions!

### DISCUSSION THREAD

**Discussion by seanap** [seanap/Plex-Audiobook-Guide#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yotsimitsu** [comment]
Hi @seanap,
first thank you on this, great job! I have a question, is there a way to tag multiple books at once? I have possibly thousands, and I'm trying to find a way that doesn't include me going through every one of them one by one...
BR

**Reply by seanap** [reply]
Hey, this all depends on the tag quality of your books. There really is no standard for tagging/naming audiobooks so I've seen tons of books that have bad/wrong tags, abbreviated title tags, or no tags at all. My tags/filenames were awful so there was no way to fully automate, and so I went through possibly thousands one by one. I will say though that using the keyboard shortcuts helps immensely. My current work flow is "ctrl-shift-i" then "alt-a 1". I was able to process ~100/hr.

If your books have the ALBUM and either ALBUMARTIST or ARTIST tag then the Audnexus plex agent should be able to match 98+%. Readarr may be able to import your books and then use Completed Download Handling to rename, but I've not used this before.

Also, you have to consider that sometimes our metadata source (Audible for my mp3tag script, or Goodreads for Readarr) contains incorrect or inconsistent (ex. different series names, or genres) data. The mp3tag script scrapes the audible website so any old or weirdly formatted pages can have errors. This doesn't happen often but when you are on the scale of 10's or 100's of thousands of books there is a not insignificant amount of errors that you will come across that would be impossible to catch if fully automated.

**Reply by yotsimitsu** [reply]
That's what I thought and it makes sense. I just wanted to be lazy :D 
I'll have to bite the bullet and just do it, it will be for the best anyway.
